k8s/overlays/main_versions/prod/kustomization.yaml

```diff
@@ -33,52 +36,55 @@ images:
     newTag: 0.15.0
   - name: gridcapa-rao-runner
     newName: docker.io/farao/gridcapa-rao-runner-with-xpress
-    newTag: v1.15.0
+    newTag: v1.16.2
   - name: gridcapa-app
     newName: docker.io/farao/gridcapa-app
-    newTag: v1.10.0
+    newTag: v1.9.0
   - name: gridcapa-gateway
     newName: docker.io/farao/gridcapa-gateway
     newTag: 1.0.0
   - name: gridcapa-cse-input-data-bridge
     newName: docker.io/farao/gridcapa-data-bridge
-    newTag: v1.2.3
+    newTag: v1.2.4
   - name: gridcapa-cse-adapter
     newName: docker.io/farao/gridcapa-cse-adapter
-    newTag: v1.12.0
+    newTag: v1.13.0
   - name: gridcapa-cse-job-launcher
     newName: docker.io/farao/gridcapa-job-launcher
     newTag: v1.6.0
   - name: gridcapa-cse-rao-logs-dispatcher
     newName: docker.io/farao/gridcapa-rao-logs-dispatcher
-    newTag: v1.1.0
+    newTag: v1.3.1
   - name: gridcapa-cse-import-runner
     newName: docker.io/farao/gridcapa-cse-cc-import-runner
-    newTag: v1.19.2
+    newTag: v1.20.0
+  - name: gridcapa-cse-export-runner
+    newName: docker.io/farao/gridcapa-cse-cc-export-runner
+    newTag: v1.20.0
   - name: gridcapa-cse-task-manager
     newName: docker.io/farao/gridcapa-task-manager
-    newTag: v1.21.1
+    newTag: v1.19.0
```

Comment:
Please check that the configmap does not change between 1.21.1 and 1.19.0

Reply:
to be checked

Reply:
checked and changed on this version